Modules/Core/Common/test/itkMatrixGTest.cxx

```diff
@@ -108,3 +135,10 @@ TEST(Matrix, GetIdentity)
   Expect_GetIdentity_returns_identity_matrix<itk::Matrix<double>>();
   Expect_GetIdentity_returns_identity_matrix<itk::Matrix<double, 2, 2>>();
 }
+
+
+TEST(Matrix, IsConstructibleFromRawArrayOfArrays)
+{
+  Expect_Matrix_is_constructible_from_raw_array_of_arrays<itk::Matrix<float>>();
```

Comment:
What are the numbers of rows and columns in this case?

Reply:
> What are the numbers of rows and columns in this case?

https://github.com/InsightSoftwareConsortium/ITK/blob/0e1c2c3dcd96fe85cd5a6d190239af950acac851/Modules/Core/Common/include/itkMatrix.h#L50-L51

My intention was to test both a default and a non-default combination of dimensions. Rather than exhaustively test all possibilities  😺